Sources/TSPL/TSPL.docc/LanguageGuide/AdvancedOperators.md

```diff
@@ -204,78 +204,84 @@ let greenComponent = (pink & 0x00FF00) >> 8   // greenComponent is 0x66, or 102
 let blueComponent = pink & 0x0000FF           // blueComponent is 0x99, or 153
 '''
 
-#### Shifting Behavior for Signed Integers
+#### Comportamento de Deslocamento para Inteiros com Sinal
 
-The shifting behavior is more complex for signed integers than for unsigned integers,
-because of the way signed integers are represented in binary.
-(The examples below are based on 8-bit signed integers for simplicity,
-but the same principles apply for signed integers of any size.)
+O comportamento de deslocamento é mais complicado para inteiros com sinal do que para inteiros sem sinal,
+por conta da maneira que são representados em modo binário.
+(Os exemplos a seguir são baseados em inteiros com sinal de 8-bits para simplificar,
+mas os mesmos princípios podem ser aplicados para inteiros com sinal de qualquer tamanho.)
 
+Inteiros com sinal usam o primeiro bit (conhecido como *bit de sinal*)
+para indicar que o inteiro é positivo ou negativo.
+O sinal para o bit `0`siginifica positivo, e o sinal para bit `1` significa negativo. 
 Signed integers use their first bit (known as the *sign bit*)
```

Comment:
acabou ficando uma frase em inglês aqui.

~Signed integers use their first bit (known as the *sign bit*)~